slowapi/extension.py

```diff
@@ -547,13 +551,13 @@ def _check_request_limit(
         Determine if the request is within limits
         """
         endpoint = request["path"] or ""
-        # view_func = current_app.view_functions.get(endpoint, None)
         view_func = endpoint_func
 
         name = "%s.%s" % (view_func.__module__, view_func.__name__) if view_func else ""
+        _endpoint_key = endpoint if self._key_style == "url" else name
```

Comment:
I guess this is made obvious by your changes, but the variable names here are a bit confusing.
Would something like `endpoint_func_name` and `endpoint_url` instead of `name` and `endpoint` be better?

Reply:
you're right, might be clearer like that